tests/acceptance/01_vars/04_containers/inline_json.cf

```diff
@@ -0,0 +1,70 @@
+#######################################################
+#
+# Test inline JSON and YAML data
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent test
+{
+  vars:
+      "options1" data => '
+{
+  "max_content": 512,
+  "verbose": 0,
+  "headers": [ "Foo: bar" ]
+}';
+
+      "options2" data => '---
+a: b
+c: d
+e:
+  - 1
+  - 2
+';
+
+      "options3" data => '
+{
+  "max_content": 512,
+  "verbose": "$(sys.os)",
```

Comment:
This will fail on all non-Linux platforms. In general it's not a good idea to use ``$(sys.os)`` in acceptance tests, better to use some local variable you have defined.